server/src/schemas/beer.js

```diff
@@ -3,9 +3,19 @@ import mongoose from "mongoose";
 const Schema = mongoose.Schema;
 
 const beerSchema = new Schema({
-  brewery: String,
-  name: String,
-  rating: Number
+  brewery: {
+    type: String,
+    required: true
+  },
+  name: {
+    type: String,
+    required: true
+  },
+
```

Comment:
Please remove this space to make the spacing consistent

Reply:
Space removed. I just thought auto format would correct It, sorry.